### DISCUSSION TITLE
Enabling Z Offset Wizard

### DISCUSSION THREAD

**Discussion by Gil80** [Jyers/Marlin#1217] [Q&A] [answered]
I tried enabling Z-offset wizard but I wasn't successful.
Am I missing anything?

`    //#define PROBE_OFFSET_WIZARD
    #if ENABLED(PROBE_OFFSET_WIZARD)
      //
      // Enable to init the Probe Z-Offset when starting the Wizard.
      // Use a height slightly above the estimated nozzle-to-probe Z offset.
      // For example, with an offset of -5, consider a starting height of -4.
      //
      //#define PROBE_OFFSET_WIZARD_START_Z -4.0

      // Set a convenient position to do the calibration (probing point and nozzle/bed-distance)
      //#define PROBE_OFFSET_WIZARD_XY_POS { X_CENTER, Y_CENTER }
    #endif
  #endif`

**Comment by Jyers** [comment, marked as answer]
You don't need to enable the offset wizard as it is enabled by default without the configuration setting.
